### PR TITLE
 ctl: replace gc_safepoint call with PD HTTP SDK

### PR DESCRIPTION
**Pull request by JackL9u**
### What problem does this PR solve?

Issue Number: Ref #7300

### What is changed and how does it work?

```commit-message
1) changed the version of the package github.com/breeswish/gin-jwt/v2 from v2.6.4-jwt-patch to v2.6.5-0.20211112114546-24b7fddb24dc to successfully compile the project. 
2) replace gc_safepoint call with PD HTTP SDK
```

### CODE REVIEW

File: client/http/api.go [hunk 1]
```diff
@@ -79,6 +79,7 @@ const (
 	Status              = "/pd/api/v1/status"
 	Version             = "/pd/api/v1/version"
 	operators           = "/pd/api/v1/operators"
+	safepoint			= "pd/api/v1/gc/safepoint"
```
Comment (JmPotato) [inline client/http/api.go:82]: Please align with other definitions:

```suggestion
	safepoint			= "/pd/api/v1/gc/safepoint"
```

File: client/http/interface.go [hunk 2]
```diff
@@ -1024,3 +1026,29 @@ func (c *client) GetKeyspaceMetaByName(ctx context.Context, keyspaceName string)
 	}
 	return &keyspaceMetaPB, nil
 }
+
+func (c *client) GetGCSafePoint(ctx context.Context) (ListServiceGCSafepoint, error) {
```
Comment (JmPotato) [inline client/http/interface.go:1030]: Each exported function should contain a line of comments.

```suggestion
// GetGCSafePoint ...
func (c *client) GetGCSafePoint(ctx context.Context) (ListServiceGCSafepoint, error) {
```

File: client/http/interface.go [hunk 3]
```diff
@@ -1024,3 +1026,29 @@ func (c *client) GetKeyspaceMetaByName(ctx context.Context, keyspaceName string)
...
+			WithURI(safepoint).
+			WithMethod(http.MethodGet).
+			WithResp(&gcsafepoint))
+	if err != nil {
+		return gcsafepoint, err
+	}
+	return gcsafepoint, nil
+}
+
+func (c *client) DeleteGCSafePoint(ctx context.Context, service_id string) (string, error) {
```
Comment (JmPotato) [inline client/http/interface.go:1043]: Ditto.

File: go.mod [hunk 4]
```diff
@@ -82,7 +82,8 @@ require (
 	github.com/aws/smithy-go v1.13.5 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bitly/go-simplejson v0.5.0 // indirect
-	github.com/breeswish/gin-jwt/v2 v2.6.4-jwt-patch // indirect
+	// github.com/breeswish/gin-jwt/v2 v2.6.4-jwt-patch // indirect
+	github.com/breeswish/gin-jwt/v2 v2.6.5-0.20211112114546-24b7fddb24dc // indirect
```
Comment (JmPotato) [inline go.mod:86]: Usually, an indirect dependency should not be modified manually. Why change it?

File: tools/pd-ut/ut.go
Comment (JmPotato) [inline tools/pd-ut/ut.go:1]: It is best not to introduce unrelated changes in a PR with a clear purpose. You can open another PR to make these modifications.

Comment (JackL9u) [reply tools/pd-ut/ut.go:1]: I see, those files were added because I merged changes from upstream/master while working in the dev branch. Will pay attention in the future!

File: client/http/interface.go (hunk 2 above)
Comment (JackL9u) [reply client/http/interface.go:1030]: will do!

File: tools/pd-ut/ut.go
Comment (JmPotato) [reply tools/pd-ut/ut.go:1]: It seems like you don't merge the upstream changes correctly, which makes them become part of "your" changes. It is recommended to do this by [rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing).
